src/Microsoft.AspNetCore.HttpOverrides/ForwardedHeadersMiddleware.cs

```diff
@@ -185,7 +185,7 @@ public void ApplyForwarders(HttpContext context)
                     }
                     else if (_options.RequireHeaderSymmetry)
                     {
-                        _logger.LogDebug(4, $"Failed to parse forwarded host: {set.Host}");
+                        _logger.LogWarning(4, $"Failed to parse forwarded host: {set.Host}");
```

Comment:
Let's clarify the message too. These are symmetry failures due to missing values. No point in showing the null/empty value.